### ISSUE TITLE
Misleading information from istio_service_setup.py

### ISSUE THREAD

**Issue by treedy** [GoogleCloudPlatform/cloud-ops-sandbox#569]
In `install.sh` the command `python3 monitoring/istio_service_setup.py $project_id $CLUSTER_ZONE $service_wait` appears to timeout. I think this script is looking for the Istio services to be added to Monitoring. The screenshot shows the Istio services are there while the script is running (double checked to make sure it was the correct monitoring Workspace).

**Output**
```
⚠️ Please create a monitoring workspace for the project by clicking on the following link:
  https://console.cloud.google.com/monitoring?project=cloud-ops-sandbox-899914728
When you are done, please PRESS ENTER TO CONTINUE
Checking to make sure necessary Istio services are ready for monitoring
cartservice was not found in Cloud Monitoring. There may be errors while provisioning Monitoring examples.
shippingservice was not found in Cloud Monitoring. There may be errors while provisioning Monitoring examples.
paymentservice was not found in Cloud Monitoring. There may be errors while provisioning Monitoring examples.
recommendationservice was not found in Cloud Monitoring. There may be errors while provisioning Monitoring examples.
checkoutservice was not found in Cloud Monitoring. There may be errors while provisioning Monitoring examples.
emailservice was not found in Cloud Monitoring. There may be errors while provisioning Monitoring examples.
productcatalogservice was not found in Cloud Monitoring. There may be errors while provisioning Monitoring examples.
frontend was not found in Cloud Monitoring. There may be errors while provisioning Monitoring examples.
currencyservice was not found in Cloud Monitoring. There may be errors while provisioning Monitoring examples.
adservice was not found in Cloud Monitoring. There may be errors while provisioning Monitoring examples.
Creating monitoring examples (dashboards, uptime checks, alerting policies, etc.)...
~/cloudshell_open/cloud-ops-sandbox/terraform/monitoring ~/cloudshell_open/cloud-ops-sandbox/terraform
```
![Screen Shot 2020-11-16 at 3 29 45 PM](https://user-images.githubusercontent.com/1851066/99305789-41a0fa00-2822-11eb-9748-69e7dd28db50.png)

**Comment by Daniel-Sanche** [comment]
Are you seeing this issue using the latest release, or deploying from master? I saw something similar recently, but I thought I fixed the issue: https://github.com/GoogleCloudPlatform/cloud-ops-sandbox/pull/563/files

**Comment by treedy** [comment]
It's been a few days, and I can't recall. I tried today on master today, but I can't even get to that step. Running into another issue which I'll file shortly.

**Comment by treedy** [comment]
I'm blocked on getting more info with #576 which I've self-assigned.

**Comment by treedy** [comment]
Master looks like it doesn't have this particular issue. Closing.
